src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/SourceEditModePageObject.java

```diff
@@ -97,7 +106,8 @@ public void checkSourceContent(String desiredContent) {
   }
 
   public void checkSourceVideoContent(String desiredContent) {
-    Assertion.assertEquals(getSourceContent().substring(1, 38) + getSourceContent().substring(48), desiredContent.substring(1, 38) + desiredContent.substring(48)
+    Assertion.assertEquals(getSourceContent().substring(1, 38) + getSourceContent().substring(48),
```

Comment:
This method needs a comment describing the root behind number magic happening inside